source/Engine.cpp

```diff
@@ -193,6 +193,11 @@ namespace {
 
 	const double RADAR_SCALE = .025;
 	const double MAX_FUEL_DISPLAY = 5000.;
+	const double ICON_SIZE = 30.;
+	const double AMMO_WIDTH = 80.;
```

Comment:
These belong in the interface definition file.

Reply:
e4f8188